lua/rainbow-delimiters/health.lua

```diff
@@ -168,7 +190,15 @@ function M.check()
 	if queries then
 		start 'Custom queries'
 		for lang, query in pairs(queries) do
-			if lang ~= '' then
+			if lang == '' then
+				if query ~= 'rainbow-delimiters' then
+					local msg = "Unknown custom default query"
+					warn(msg)
+				else
+					local msg = "Valid custom default query"
+					ok(msg)
+				end
```

Comment:
I am not sure about this. If someone defines his own custom query for all the languages he uses, then that's perfectly valid. Otherwise there is no real point of having an option for `''` in the first place if only one fixed value passes the test.

Reply:
This is the reason I made it a warning instead of an error and wrote "unknown" instead of "invalid", but it might be better to just remove this. I put it there to catch spelling mistakes, but the LSP should already help with that.

Reply:
I tried rewriting it a bit now and return ok instead of warning. Please let me know if the updated version is fine or if you prefer to just remove this? I think it makes sense to include a check here since a lot of users probably write out the default setup from the docs.

I can squash the commits when I know what to do here.